client/packages/system/src/Item/DetailView/DetailView.tsx

```diff
@@ -30,6 +32,53 @@ export const ItemDetailView: FC = () => {
 
   if (isLoading || !data) return <DetailFormSkeleton />;
 
+  const onLedgerRowClick = (ledger: ItemLedgerFragment) => {
+    switch (ledger.invoiceType) {
+      case InvoiceNodeType.InboundShipment:
+        navigate(
+          RouteBuilder.create(AppRoute.Replenishment)
+            .addPart(AppRoute.InboundShipment)
+            .addPart(String(ledger.invoiceNumber))
+            .build()
+        );
+        break;
+      case InvoiceNodeType.SupplierReturn:
+        navigate(
+          RouteBuilder.create(AppRoute.Replenishment)
+            .addPart(AppRoute.SupplierReturn)
+            .addPart(String(ledger.invoiceNumber))
+            .build()
+        );
+        break;
+      case InvoiceNodeType.OutboundShipment:
+        navigate(
+          RouteBuilder.create(AppRoute.Distribution)
+            .addPart(AppRoute.OutboundShipment)
+            .addPart(String(ledger.invoiceNumber))
+            .build()
+        );
+        break;
+      case InvoiceNodeType.CustomerReturn:
+        navigate(
+          RouteBuilder.create(AppRoute.Distribution)
+            .addPart(AppRoute.CustomerReturn)
+            .addPart(String(ledger.invoiceNumber))
+            .build()
+        );
+        break;
+      case InvoiceNodeType.Prescription:
+        navigate(
+          RouteBuilder.create(AppRoute.Dispensary)
+            .addPart(AppRoute.Prescription)
+            .addPart(String(ledger.invoiceNumber))
+            .build()
+        );
+        break;
+      default:
+        break;
+    }
```

Comment:
I'm wondering if toast / snack would be helpful where no navigation is available for inventory adjustments if it isn't obvious for users.

Reply:
I think okay for now. Something to note in the future maybe. This was a bonus and isn't in the specs XD

Reply:
Just not being able to click is fine. This was an added bonus, I don't know if they will actually use it